### PR TITLE
Feature/aa 220 add notifications via notification center for administrator

### PR DESCRIPTION
**Pull request by AlexeyShok**
Added notifications for administrators via NotificationCenter. Added notification type to ReceveMessage method

### CODE REVIEW

File: Api/NotificationCenter/Hubs/INotificationClient.cs [hunk 1]
```diff
@@ -5,6 +5,6 @@ namespace HappyTravel.Edo.Api.NotificationCenter.Hubs
 {
     public interface INotificationClient
     {
-        Task ReceiveMessage(int messageId, JsonDocument message);
+        Task ReceiveMessage(int messageId, string notificationType, JsonDocument message);
```
Comment (kirillta) [inline Api/NotificationCenter/Hubs/INotificationClient.cs:8]: Why string?

File: Api/Services/Accommodations/Bookings/Management/BookingRecordsUpdater.cs [hunk 2]
```diff
@@ -29,22 +29,22 @@ public class BookingRecordsUpdater : IBookingRecordsUpdater
     {
         public BookingRecordsUpdater(IDateTimeProvider dateTimeProvider,
             IBookingInfoService infoService,
-            IBookingNotificationService notificationService,
+            IBookingNotificationService bookingNotificationService,
```
Comment (kirillta) [inline Api/Services/Accommodations/Bookings/Management/BookingRecordsUpdater.cs:32]: Please, place input variables in a more compact way

File: HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs [hunk 3]
```diff
@@ -16,10 +16,21 @@ public enum NotificationTypes
         CreditCardPaymentReceived = 9,
         AccountBalanceReplenished = 10,
         // Counterparty
-        CustomerInvitation = 11,
+        RegularCustomerInvitation = 11,
```
Comment (kirillta) [inline HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs:19]: Who the _regular_ customer is?

File: Api/NotificationCenter/Services/NotificationService.cs [hunk 4]
```diff
@@ -111,6 +111,12 @@ public async Task<Result> Send(DataWithCompanyInfo messageData, NotificationType
         }
 
 
+        public async Task<Result> Send(DataWithCompanyInfo messageData, NotificationTypes notificationType, List<string> emails, string templateId)
+        {
+            return await Send(new SlimAdminContext(adminId: 0), messageData, notificationType, emails, templateId);
```
Comment (oleg1std) [inline Api/NotificationCenter/Services/NotificationService.cs:116]: lambda operator => can be used here

File: HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs (hunk 3 above)
Comment (AlexeyShok) [reply HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs:19]: We have a regular agent and a master agent. They have different registration flows and different forms of the letter template and its recipients. As an option, I suggest calling RegularAgentInvitation. This more accurately reflects the meaning.

File: Api/NotificationCenter/Hubs/INotificationClient.cs (hunk 1 above)
Comment (AlexeyShok) [reply Api/NotificationCenter/Hubs/INotificationClient.cs:8]: When transmitted in JsonDocument format, data is lost, only the header is transmitted.

File: Api/NotificationCenter/Services/NotificationService.cs (hunk 4 above)
Comment (AlexeyShok) [reply Api/NotificationCenter/Services/NotificationService.cs]: Fixed

File: Api/Services/Accommodations/Bookings/Management/BookingRecordsUpdater.cs (hunk 2 above)
Comment (AlexeyShok) [reply Api/Services/Accommodations/Bookings/Management/BookingRecordsUpdater.cs]: Fixed

File: HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs (hunk 3 above)
Comment (AlexeyShok) [reply HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs]: Renamed to AgentInvitation

**Review by kirillta — APPROVED** [review]

**Pull request merged**